packages/asset/test/AssetCreate.test.ts

```diff
@@ -1142,4 +1142,200 @@ describe('AssetCreate (/packages/asset/contracts/AssetCreate.sol)', function ()
       });
     });
   });
+  describe('Token info', function () {
```

Comment:
This section should not be in the AssetCreate test file, since the functions are in Asset.sol
They should be in the Asset test file.
Please also fix the naming so it is clear that this relates to use of the TokenUtils lib in Asset.

Reply:
We have to checkout the right information regarding the tier and creator etc which are supplied via the AssetCreate contract. That is why I wrote test cases in AssetCreate test file

Reply:
Understood. 
However, this seems to be a "quick fix" approach - it would be good for readability to have these in the Asset test file and use a dummy Asset fixture.
OK to leave as-is for the time being.